jenkins/Pipeline.gy

```diff
@@ -6,13 +6,19 @@ pipeline {
         IMAGE_REPO="quay.io/pbench"
         IMAGE_ROLE="ci"
         NO_COLORS=0
-        PB_EXTERNAL_CONTAINER_REG=credentials('24a93506-ecd6-403a-b4f0-386f9cc943e9')
-        PB_INTERNAL_CONTAINER_REG=credentials('c3e2d737-0e56-4c1e-a945-d86bc644384c')
-        PODMAN_EXTERNAL=credentials('87ad2797-02eb-464f-989f-8ab78d63cdf3')
-        PODMAN_INTERNAL=credentials('12b404ca-3036-4960-9929-979148b9e49a')
         PY_COLORS=0
         TERM='dumb'
 
+        PODMAN_INTERNAL=credentials('12b404ca-3036-4960-9929-979148b9e49a')
+
+        PB_CONTAINER_REG=credentials('c3e2d737-0e56-4c1e-a945-d86bc644384c')
```

Comment:
Perhaps.

I don't think this is an internal vs external container registry.  It is just "the" container registry to use.  So for Server containers we'd use one, and for agent we'd use another, but we don't necessarily need two different variables.

Reply:
Under the slightly bold assumption that the same pipeline will be building _both_ Pbench Server **and** Agent containers, we'll need to have definitions for the registries for the Server container(s) and the Agent containers, respectively.  Now, if you like, we can style them as "Server registry" and "Agent registry" rather than "internal registry" and "external registry", but...unless we're counting on putting the Server images and the Agent images in the _same_ registry, we will need two separate definitions, once the CI starts building the Agent containers (which is, like, the next PR after #3086).

Reply:
Yeah, OK; I get what Webb's saying. Each "build step" only cares about one repo, but right now the CI needs to build *both*. So while the `Pipeline.gy` can supply a different value of a common parameter to the container build script, Jenkins needs both available.

An alternative might be that Jenkins always builds to `images.paas.redhat.com` and we perform a separate "release" step to copy the agent containers to `copr`.

Reply:
Work pulled out into PR #3112.